packages/generator/templates/validations/schemas.ts

```diff
@@ -0,0 +1,20 @@
+import {z} from "zod"
+
+if (process.env.parentModel) {
+    export const __ModelName__Schema = z.object({
+      __parentModelId__: z.__parentModelIdZodType__(),
+      // template: __fieldName__: z.__zodType__(),
+    })
+} else {
+    export const __ModelName__Schema = z.object({
```

Comment:
plus update where this is used
```suggestion
    export const Create__ModelName__Schema = z.object({
      __parentModelId__: z.__parentModelIdZodType__(),
      // template: __fieldName__: z.__zodType__(),
    })
} else {
    export const Create__ModelName__Schema = z.object({
```